src/main/java/org/wiremock/integrations/testcontainers/WireMockContainer.java

```diff
@@ -85,6 +77,9 @@ public WireMockContainer(String image, String version) {
      */
     public WireMockContainer withCliArg(String arg) {
         //TODO: Switch to framework with proper CLI escaping
+        if (arg.contains("--verbose")) {
```

Comment:
Could we just add an explicit method like `withBannerDisabled`() ? While reasonable to add only in verbose logging, it is not aligned with the current WireMock CLI behavior

Reply:
If I got the issue right [ 31 ](https://github.com/wiremock/wiremock-testcontainers-java/issues/31), the default behaviour should be starting without the banner so I could add the method withBanner instead of withBannerDisabled right? 
But you are right then the behaviour is not aligned with the WireMock CLI default behaviour, where it starts by default with the banner.